src/components/Company/Offers/Manage/CollapsedQuickOfferEdit.js

```diff
@@ -0,0 +1,102 @@
+import React, { useCallback, useContext } from "react";
+import PropTypes from "prop-types";
+import PublicationEndDateComponent from "../../../Offers/Form/form-components/PublicationEndDateComponent";
+import EditOfferSchema from "../../../Offers/Edit/EditOfferSchema";
+import { Box, Button, Typography } from "@material-ui/core";
+import useOfferForm from "../../../../hooks/useOfferForm";
+import { editOffer } from "../../../../services/offerService";
+import { parseRequestErrors } from "../../../Offers/Form/OfferUtils";
+import { useDispatch } from "react-redux";
+import { addSnackbar } from "../../../../actions/notificationActions";
+import { OfferManagementContext } from "./CompanyOffersManagementWidget";
+
+const CollapsedQuickOfferEdit = ({ offerId }) => {
+    const dispatch = useDispatch();
+    const dispatchAddSnackbarAction = useCallback(
+        (notification) => {
+            dispatch(addSnackbar(notification));
+        },
+        [dispatch]
+    );
+
+    const {
+        offers,
+        setOffers,
+    } = useContext(OfferManagementContext);
+
+    const { publishEndDate } = offers[offerId]["fields"];
+
+    const { control, fields, errors } = useOfferForm(EditOfferSchema, {
+        publishEndDate: publishEndDate.value,
+    });
+
+    const isObjectEmpty = useCallback(
+        (object) =>
+            Object.keys(object).length === 0 &&
+      Object.getPrototypeOf(object) === Object.prototype
+        , []);
+
+    const displaySuccessMessage = useCallback(() => {
+        dispatchAddSnackbarAction({
+            message: "Succesfully updated",
```

Comment:
Small typo

```suggestion
            message: "Successfully updated",
```

Reply:
Thank you, I already rebased and fixed that minor spelling mistake.